test_rclcpp/test/test_executor.cpp

```diff
@@ -61,7 +61,7 @@ TEST(CLASSNAME(test_executor, RMW_IMPLEMENTATION), spin_some_max_duration) {
   rclcpp::executors::SingleThreadedExecutor executor;
   auto node = rclcpp::Node::make_shared("spin_some_max_duration");
   auto lambda = []() {
-      std::this_thread::sleep_for(1ms);
+      std::this_thread::sleep_for(100ms);
```

Comment:
This is a good change, and keeps the current semantics of the test; basically, that 20 threads of N ms sleep take < 20*N ms to complete.  However, those semantics are a bit hard to discern here.  Would you mind adding a comment explaining that?

Reply:
There was already a comment, but I forgot to update it.
See https://github.com/ros2/system_tests/pull/430/commits/2d0e91cc77c569049844ad410df4a6fa3956b728.